internal/devconfig/configfile/ast.go

```diff
@@ -216,7 +216,72 @@ func (c *configAST) appendAllowInsecure(name, fieldName string, whitelist []stri
 	c.appendStringSliceField(name, fieldName, whitelist)
 }
 
-func (c *configAST) FindPkgObject(name string) *hujson.Object {
+// removePatch removes the patch field from the named package.
+func (c *configAST) removePatch(name string) {
```

Comment:
Could we generalize these similar to `setPackageBool` ? 

`removePackageField` for removing.
`setPackageString` for adding. 

`setPatch` could wrap these (remove `patch_glibc` add `patch`) or you could not special case at all and have the code in packages.go do that logic). That keeps the ast more logic agnostic.

Honestly, you could also not remove `patch_glibc`. It doesn't really do harm and would allow us to have simpler code.

Reply:
Making them more general sounds good to me. I'd like to do in a follow-up PR though just to get the patching stuff out the door.